CHANGELOG.md

```diff
@@ -3,7 +3,7 @@
 As this project is pre 1.0, breaking changes may happen for minor version bumps.
 A breaking change will get clearly marked in this log.
 
-## In `master`
+## [v0.14.0](https://github.com/stellar/js-stellar-sdk/compare/v0.13.0...v0.14.0)
```

Comment:
Can you add info about new functions in Horizon 0.17.0 (1eb6ad7)? I forgot to add it to CHANGELOG, sorry!

Reply:
@bartekn Check it out now, see if my notes are accurate.